ic-agent/src/agent/response.rs

```diff
@@ -1,3 +1,5 @@
+use super::replica_api::RejectResponse;
```

Comment:
nit: for consistency I think you should use 

"use crate::..."

Reply:
Solved in #429